networking/kvm.go

```diff
@@ -119,6 +120,11 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf) (netlink.Link,
 	if err := netlink.LinkAdd(link); err != nil {
 		return nil, errwrap.Wrap(errors.New("cannot create macvtap interface"), err)
 	}
+	if err := netlink.LinkSetUp(link); err != nil {
+		// remove newly added link ignoring errors, we are already in errornous situation
+		netlink.LinkDel(link)
```

Comment:
So this returns an error? Then probably `_ = netlink.LinkDel(link)` would be better.